core/src/utils/test/overlays/overlays.e2e.ts

```diff
@@ -255,4 +302,65 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       await expect(modalInputOne).toBeFocused();
     });
   });
+
+  test.describe(title('overlays: create'), () => {
```

Comment:
`mouse.wheel` doesn't wait for scrolling to finish before returning (https://playwright.dev/docs/api/class-mouse#mouse-wheel) so this might be flaky. As an alternative (and potential spec test) are we able to check the following:

1. Root does not have `overflow: hidden` evaluated styles when overlay is hidden
2. Root does have `overflow: hidden` evaluated styles when overlay is presented

Reply:
I'm not sure if this is much better, but using https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle could help. At the very least it would avoid any scrolling flakiness.

Reply:
Good thoughts! I'll try them out!